### DISCUSSION TITLE
Birdweather number

### DISCUSSION THREAD

**Discussion by noiselevelzero** [mcguirepr89/BirdNET-Pi#1187] [General]
Hi,  new to Birdnet Pi and really enjoying it!  The discussions have been a great education.

Is Birdweather still accepting requests for ID's?  I have sent at least 5 e-mails, including two to the new address that Tim of Birdweather asked us to start using, and have not received a reply in at least a week.  Is there something else I need to do to get the ID besides the info generated by the app?

Thank you for any information.

**Comment by dwxt** [comment]
Same question here, I have sent 3 mails to Tim, and waiting for reply.

**Comment by Svardsten53** [comment]
I have sent a reminder to Tim in another forum.

**Comment by timsterc** [comment]
I've been slammed the past week+   Lots of things to keep up with for our small team.  

Appreciate your patience.

**Reply by dwxt** [reply]
Hi Tim, Thank you for your reply. Take your time! wish you all the best

**Comment by noiselevelzero** [comment]
Thank you Tim.  Completely understand.  I wanted to make sure there was not some other requirement I'd not yet fulfilled for a number.  Please only process the most recent note I sent to you on your new E-mail for this purpose and ignore all others.

Thank you.
